package.json

```diff
@@ -27,6 +27,7 @@
         "url": "git+ssh://git@github.com/datawrapper/shared.git"
     },
     "devDependencies": {
+        "@datawrapper/chart-core": "8.30",
```

Comment:
I don't see `chart-core` being imported anywhere in this PR so perhaps we can remove it from the dev dependencies.

Reply:
you're right :+1: